pom.xml

```diff
@@ -230,13 +235,11 @@ Jean-Yves Tinevez and Michael Zinsmaier.</license.copyrightOwners>
 		<dependency>
 			<groupId>org.openjdk.jmh</groupId>
 			<artifactId>jmh-core</artifactId>
-			<version>1.19</version>
```

Comment:
Noting that the jmh version can be removed the version is now specified in the scijava pom:
https://github.com/scijava/pom-scijava/commit/2f29fb81acd5b33097ff80b8ab3af4cd33fcbd7f

Merging this would result in jmh being updated to version 1.23
https://github.com/scijava/pom-scijava/blob/7847a06b47b12ae200f70588c4cf573ed3d74e57/pom.xml#L1084

Reply:
Is there any particular reason why `imglib2` still needs to stick to version 1.19? In this case the version override should be done via version `<property>` and not hard-coding. But I think it's fine leaving this to be managed by pom-scijava, no?

Reply:
yes, should be managed by pom-scijava 👍